sherpa/astro/xspec/__init__.py

```diff
@@ -219,6 +219,39 @@ def get_xsabundances() -> dict[str, float]:
             for name in get_xselements().keys()}
 
 
+def get_xsabund_doc(name: Optional[str] = None) -> str:
+    """Return the documentation for the abundance table.
+
+    Parameters
+    ----------
+    name : str or None, optional
+        If not set, use the current abundance table
+
+    Returns
+    -------
+    doc : str
+        The documentation for the table
+
+    See Also
+    --------
+    get_xsabund
+
+    Examples
+    --------
+
+    >>> get_xsabund_doc("angr")
+    'Anders E. & Grevesse N. Geochimica et Cosmochimica Acta 53, 197 (1989)'
+
+    """
+
+    aname = get_xsabund() if name is None else name
+
+    # Looks the return value includes a leading space so remove it.
```

Comment:
the strip() command in 252 removes leading/trailing white space. Perhaps change comment to "Remove and leading or trailing white space".  Alternative, use lstrip() to remove leading space.